tests/unit/amo/components/TestHeader.js

```diff
@@ -178,4 +178,17 @@ describe(__filename, () => {
 
     expect(link).toHaveLength(0);
   });
+
+  it('displays manage my submissions when user is logged in', () => {
+    const username = 'championshuttler';
+    const { store } = dispatchSignInActions({
+      userProps: { username },
+    });
+    const wrapper = renderHeader({ store });
+    const link = wrapper.find('.Header-user-menu-developers-submissions-on-link');
```

Comment:
You need to find "something" that exists. Here you use a CSS class name, which is nice but it does not exist in the `Header` component. Add a `className` prop to the `Link` and it should work.

Reply:
@championshuttler do you need more information? I think you almost got it right.

Reply:
@willdurand I was searching for it that how to add it and also tried in test cases writtern already but did not get any lead . Can you guide me how to do that?

Reply:
What if you add a `className` prop to the link you've added? Its value will be `Header-user-menu-developers-submissions-on-link`

Reply:
@willdurand I updated it according to this airbnb.io/enzyme/docs/api/ReactWrapper/find.html# . But its failing again with error `Failed to parse selector: [className = "Header-user-menu-developers-submissions-on-link"]`

Reply:
Yes, the previous version of your code was ok: `wrapper.find('.Header-user-menu-developers-submissions-on-link')`